walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +272,112 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendees1() {
```

Comment:
can you use test names that give an idea of what the test is about? e.g. "testOverlapRequiredAttendees" or something

Reply:
One approach we use a lot at Google is to make all the test names predicates, by which I mean a sentence fragment that could complete the sentence that begins "We are testing to make sure that it...".  So this could be something like `testAllowsConflictsForOptionalAttendees`